src/mixins/nft.js

```diff
@@ -592,9 +593,8 @@ export default {
     async fetchISCNMetadata() {
       await this.lazyGetISCNMetadataById(this.iscnId);
     },
-    async updateNFTClassMetadata() {
-      await catchAxiosError(this.fetchNFTClassMetadata(this.classId));
-      this.lazyGetUserInfoByAddresses(this.iscnOwner);
+    async updateNFTClassAggregatedInfo() {
+      await catchAxiosError(this.fetchNFTClassAggregatedInfo(this.classId));
```

Comment:
Do we need to call `lazyGetUserInfoByAddresses` at some point? Maybe keep `updateNFTClassMetadata` and call `updateNFTClassAggregatedInfo` and `lazyGetUserInfoByAddresses`?

Reply:
Currently `fetchNFTClassAggregatedInfo` always calls `lazyGetUserInfoByAddresses` here:
https://github.com/likecoin/liker-land/pull/1168/files#diff-bc31414c5136bf8f64828fe6722217f2a2dc6a0e66fc238cce705547403f6dc2R423-R426

Is it preferred not to call `lazyGetUserInfoByAddresses` inside `fetchNFTClassAggregatedInfo` but combine them in `updateNFTClassMetadata`?

Reply:
Oh just keep it the current way then